### ISSUE TITLE
redmine - passenger phusion download link is outdated

### ISSUE THREAD

**Issue by r10r** [bitnami/charts#7220]
**Which chart**:
redmine-17.0.2

**Describe the bug**

The download for passenger phusion is outdated.

Current versions are available here https://github.com/phusion/passenger/releases/

Output from `kubectl logs -n redmine pod/redmine-56ff5cbd7b-npvth`:

```
[passenger_native_support.so] trying to compile for the current user (root) and Ruby interpreter...
     (set PASSENGER_COMPILE_NATIVE_SUPPORT_BINARY=0 to disable)
     Warning: compilation didn't succeed. To learn why, read this file:
     /tmp/passenger_native_support-czomcv.log
 [passenger_native_support.so] finding downloads for the current Ruby interpreter...
     (set PASSENGER_DOWNLOAD_NATIVE_SUPPORT_BINARY=0 to disable)
     Could not download https://github.com/phusion/passenger/releases/download/release-6.0.10/rubyext-ruby-2.6.8-x86_64-linux.tar.gz: The requested URL returned error: 404 
     Trying next mirror...
     Could not download https://oss-binaries.phusionpassenger.com/binaries/passenger/by_release/6.0.10/rubyext-ruby-2.6.8-x86_64-linux.tar.gz: The requested URL returned error: 404 
     Trying next mirror...
     Could not download https://s3.amazonaws.com/phusion-passenger/binaries/passenger/by_release/6.0.10/rubyext-ruby-2.6.8-x86_64-linux.tar.gz: The requested URL returned error: 403 Forbidden
 [passenger_native_support.so] will not be used (can't compile or download) 
  --> Passenger will still operate normally.
=============== Phusion Passenger(R) Standalone web server started ===============
PID file: /opt/bitnami/redmine/passenger.3000.pid
Log file: /opt/bitnami/redmine/log/passenger.3000.log
Environment: production
Accessible via: http://0.0.0.0:3000/

You can stop Phusion Passenger(R) Standalone by pressing Ctrl-C.
Problems? Check https://www.phusionpassenger.com/library/admin/standalone/troubleshooting/
===============================================================================
[ N 2021-08-13 12:07:14.7120 306/T5 age/Cor/SecurityUpdateChecker.h:519 ]: Security update check: no update found (next check in 24 hours)
App 345 output:  [passenger_native_support.so] trying to compile for the current user (redmine) and Ruby interpreter...
App 345 output:      (set PASSENGER_COMPILE_NATIVE_SUPPORT_BINARY=0 to disable)
App 345 output:      Warning: compilation didn't succeed. To learn why, read this file:
App 345 output:      /tmp/passenger_native_support-111lu5u.log
App 345 output:  [passenger_native_support.so] finding downloads for the current Ruby interpreter...
App 345 output:      (set PASSENGER_DOWNLOAD_NATIVE_SUPPORT_BINARY=0 to disable)
App 345 output:      Could not download https://github.com/phusion/passenger/releases/download/release-6.0.10/rubyext-ruby-2.6.8-x86_64-linux.tar.gz: The requested URL returned error: 404 
App 345 output:      Trying next mirror...
App 345 output:      Could not download https://oss-binaries.phusionpassenger.com/binaries/passenger/by_release/6.0.10/rubyext-ruby-2.6.8-x86_64-linux.tar.gz: The requested URL returned error: 404 
App 345 output:      Trying next mirror...
App 345 output:      Could not download https://s3.amazonaws.com/phusion-passenger/binaries/passenger/by_release/6.0.10/rubyext-ruby-2.6.8-x86_64-linux.tar.gz: The requested URL returned error: 403 Forbidden
App 345 output:  [passenger_native_support.so] will not be used (can't compile or download) 
App 345 output:   --> Passenger will still operate normally.
```

**Comment by r10r** [comment]
I think it's the rubygem `passenger-6.0.10` that should be updated.

**Comment by marcosbc** [comment]
Hi @r10r, thanks for reporting this. We are aware of issues like these, they happen when the bundled Phusion Passenger version is built for a different Ruby version than the one bundled in the image.

I've started a rebuild of the Redmine container image with the fixed Passenger package version, we expect to have it released during the next couple of days.

**Comment by r10r** [comment]
Thanks @marcosbc ! So simply rebuilding the container image fixes this ? Or is there any further modification required ?

**Comment by marcosbc** [comment]
Unfortunately it requires to run the internal scripts to rebuild the Redmine tarball, which is done with our internal pipeline. This tarball will include the rebuilt Passenger gem with support for the latest Ruby version used in the Redmine container.

**Comment by r10r** [comment]
@marcosb Is this already fixed ?

**Comment by marcosbc** [comment]
Hi @r10r, yes this should be fixed in the current version of the chart, `17.0.7`. The Passenger native library is compiled for Ruby 2.6.8, which is the version included in this image.

**Comment by r10r** [comment]
Thanks! I'll upgrade the chart soon.
